spec/formatterSpec.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+var formatDebitTransaction = require('../src/formatter').formatDebitTransaction
+var formatCreditTransaction = require('../src/formatter').formatCreditTransaction
+var sortedTransactionsByDateAscending = require('../src/formatter').sortedTransactionsByDateAscending;
+var addBalanceToEachTransaction = require('../src/formatter').addBalanceToEachTransaction;
+var revertDateToString = require('../src/formatter').revertDateToString;
+var convertStringToDate = require('../src/transaction').convertStringToDate
+
+describe('formatter functions', function () {
+  var listOfTransactionsMock, listOfTransactionsMockSorted, date;
+  beforeEach(function () {
+    listOfTransactionsMock = [
+      {
+        date: convertStringToDate('13/01/2012'),
+        amount: 2000 },
+      {
+        date: convertStringToDate('14/01/2012'),
+        amount: -500 },
+      {
+        date: convertStringToDate('10/01/2012'),
+        amount: 1000 } ]
+  });
+  describe('#formatDebitTransaction', function () {
+    it('generates a string with debit amount in the thirds column', function () {
+      var transaction = {
+        date: '10/01/2012',
+        amount: -1000,
+        balance: 1000}
+      expect(formatDebitTransaction(transaction)).toBe(' || || 1000 || 1000')
```

Comment:
Like above, methods like formatDebitTransaction feel like private methods and so ones not to test.